docs/src/schemas/experiment_metadata.md

```diff
@@ -1,43 +1,70 @@
 (readmeexp)=
 # Experiment metadata
 
-|key | description |
-| ----| -----|
-|DOI | DOI of the of the original publication of the experimental data|
-TEMPERATURE | Temperature of the experiment
-MOLAR_FRACTIONS | Dictionary of molar fractions of bilayer components
-ION_CONCENTRATIONS | Dictionary of ion concentrations in the system
-TOTAL_LIPID_CONCENTRATION | Total concentration of lipid components
-COUNTER_IONS | Type of counter ions if present
+**Global metadata table**
+
+|             key           | description |
+|---------------------------|-----------------------------------------|
+| ARTICLE_DOI | DOI of the of the original publication of the experimental data|
+| DATA_DOI | DOI of the dataset deposition with raw NMR data |
+| TEMPERATURE | Temperature of the experiment |
+| MEMBRANE_COMPOSITION | Dictionary of molar fractions of membrane phase |
+| SOLUTION_COMPOSITION | Dictionary of ion concentrations in the system |
+| PH | pH of the system |
+| PH_METHOD | Method of pH setting or measurement (buffer / measurement) |
+| REAGENT_SOURCES | Description of lipid reagents. Source, purity, etc. |
+
+**NMR specific metadata**
+
+|             key           | description                             |
+|---------------------------|-----------------------------------------|
+| T_RF_HEATING | Correction of the temperature according to RF heating |
+| TOTAL_HYDRATION | Total hydration of the system (water mass %) |
+| HYDRATION_METHOD | Method of preparing lamellar phase |
+| NMR_EXPERIMENT | Description of NMR experiment type |
 
 **Field descriptions**
 
-1. **DOI**  
+1. **ARTICLE_DOI**  
 DOI of the original publication where the experimental data originates.
 
-2. **TEMPERATURE**  
-Temperature of the experiment.
+2. **DATA_DOI**  
```

Comment:
doi is nice, but what if the dataset does not have a doi, but only some other PID or even just a URL?